datalad_next/create_sibling_webdav.py

```diff
@@ -69,6 +70,9 @@ class CreateSiblingWebDAV(Interface):
     server needs to be specified for creating a sibling. However, the sibling
     setup can be flexibly customized (no storage sibling, or only a storage
     sibling, multi-version storage, or human-browsable single-version storage).
+    The target location is currently expected by this command to have no
+    potentially conflicting content. Please make sure of that beforehand!
+    Such content may be overwritten.
```

Comment:
I think we need to be explicit here: what is "potentially conflicting content" exactly? When would it be overwritten?

Reply:
I believe the only thing that is touched on the server by this command is the root dir of each dataset (i.e. `mkdir`).

Reply:
Think so, too. Except: The git remote comes with it's own if enabled, right?

Reply:
But still: Any other content can still be in conflict with a subsequent push. And we can't detect it, if it wasn't created by another export, I think.

Reply:
So nothing will be overwritten by *this* command.

Reply:
I went for a minimal statement

> This command does not check for conflicting content on the WebDAV server!